src/containers/board/reducer.js

```diff
@@ -30,9 +32,10 @@ export default (state = initialState, action) => {
         ...state,
         ...action.payload.data,
       }
-    case `${GET_ONE_BOARD}_ERROR`:
+    case `${GET_ONE_BOARD}_FAIL`:
       return {
         ...state,
+        isFailure: true,
```

Comment:
You need to set `isFailure` to `false` for SUCCESS cases (`GET_ALL_LISTS_IN_BOARD` and `GET_ONE_BOARD`)

Reply:
done